source/process/casestudy.rst

```diff
@@ -106,20 +106,31 @@ Case Study Interview Process
 Writing Guidelines 
 ----------------------------
 
-1. Write so that people with English as a second language can understand every sentence. 
-2. Don't disparage other products. It's okay to factually address differences on why Mattermost was selected. 
+1. DO NOT use slang or jargon. Write so that people with English as a second language can understand every sentence. 
+2. DO NOT debase the competition or disparage other products. It's okay to factually address differences on why Mattermost was selected. 
```

Comment:
Wondering if a simpler word can be used than "debase"?

Reply:
How about: "DO NOT criticize the competition or disparage other products..."

Would that work?

Reply:
Sure thing! Made the edit